pkg/graph/resolvers/model_plan.go

```diff
@@ -158,6 +180,27 @@ func ModelPlanCreate(
 				), zap.Error(sendEmailErr))
 			}
 		}()
+
+		for _, emailPref := range newModelPlanEmailPrefs {
```

Comment:
Does this have to iterate through each user because you need the id for the unsubscribe link? Otherwise you could maybe do a BCC list or something.

Reply:
I have tried this both ways. In the end, whether by multiple recipients or BCC, the recipient email list is delivered to all recipients. At best, this is inconsistent UX and has a less professional presentation. At worst, there may be some privacy and informational transaction concerns. If we get into multiple thousands of recipients that may be cause for some performance concern, but for the foreseeable future we do not have significant enough scale for this to be a primary concern. At that scale we would need to employ a dispatch service like SES anyway.

In conclusion, for the purposes of consistency with other emails and privacy I will maintain this as individually sent emails rather than batching them.

Reply:
I revisited this. Clay and I did some email testing, and are confident about the behavior of a BCC approach despite the administrative view of Mailcatcher. Suggestion integrated. 👍

Reply:
Awesome, thanks for exploring that.